test/test-test.js

```diff
@@ -33,6 +34,11 @@ function read (filepath) {
     return fs.readFileSync(path.join(__dirname, filepath), 'utf8');
 }
 
+function replaceHelpers (src) {
+    // Will replace the require for helpers so we can test the component
+    return src.replace("require('svg-react-loader/helpers')", "require('../helpers')");
+}
+
```

Comment:
Without this tests we're failing. Not sure if this is the right approach though..